common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -3,56 +3,31 @@
 ===================
-
-DJI V1 FPV Goggles
-
- .. image:: ../../../images/msp_dji_fpv_goggles.jpeg
-    :target: ../_images/msp_dji_fpv_goggles.jpeg
-
- 
- DJI Goggles RE
- 
- .. image:: ../../../images/msp_dji_goggles_re.jpeg
-    :target: ../_images/msp_dji_goggles_re.jpeg
- 
-Features
---------
- 
- - ArduPilot currently supports all of the OSD panel items provided by the V1 and V2 DJI FPV Goggles, details are given in the table below.
- - Changing display units other than metric and imperial are not currently supported.
- - Multiple screens and remote switching of those screens is supported.
- - Displaying statistics on a dedicated screen is supported, see below for details.
- - Warning levels for RSSI, Voltage, etc. currently not supported
+Telemetry based OSDs will render OSD panel items on screen with their own engine, so ArduPilot has no control of how the items look. Another limitation of telemetry based OSDs is that there's no way for ArduPilot to add new panel items at will, it's the vendor's responsibility to add new features by rolling out new firmware releases.
 
 Configuration
 -------------
-
-To enable MSP OSD, set the following parameters ( example using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
+Example assumes connection of air unit to SERIAL2 of the flight controller.
 
  - :ref:`OSD_TYPE<OSD_TYPE>` = 3 if no integrated OSD is being used in order to activate the OSD code. If an integrated OSD is present and the user wishes to have both OSDs , then :ref:`OSD_TYPE<OSD_TYPE>` = 1 will activate the on-board OSD as well as providing screens for the MSP OSD function. For example, on vehicles using the DJI goggles/air system for medium range, but still running a long range VTX using the internal OSD for when the vehicle exceeds the range of the HD DJI Goggles. This configuration could use one OSD screen optimized for DJI Goggles, and another for the integrated OSD and the user can switch between them depending on which video system is being viewed.
- - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 33
- - :ref:`MSP_OPTIONS<MSP_OPTIONS>` = 0 (polling mode)
- 
-.. note:: Serial port buad rate default is changed to 115.2Kbaud automatically when setting the above protocol type. However, if the user has previously or subsequently changes the baud, this default will not be used. 115.2Kbaud is required by most video goggle systems.
+ - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 32 (MSP telemetry only mode)
+ - :ref:`SERIAL2_BAUD<SERIAL2_BAUD>` = 115
+ - :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 0 = 1 (EnableTelemetryMode), but only if the OSD is to be connected only via a single wire from its RX input to the Serial ports TX. This is known as "push" mode.
 
-.. note:: DJI Custom OSD must be enabled: in SETTINGS->DISPLAY->CUSTOM OSD menu of goggles
-
-OSD Panel Items
----------------
+ - These are the MSP only OSD elements supported on telemetry based OSDs (assuming the OSD has the capability itself).
```

Comment:
I disagree strongly with you about embedding this table here @Hwurzburg . Sure it makes sense if you have this type of goggles, but for anyone else it makes the page very hard to read and find what you are looking for. It's your call of course.

Reply:
it belongs with the MSP only stuff